draft-scenarios/metadata.tex

```diff
@@ -96,7 +96,7 @@
 \newtoggle{githubbuild}
 
 % Variables
-\def\_assets{assets}
+\def\_assets{../assets}
```

Comment:
I don't think this will compile. Are you on Windows?

Reply:
You're quick, haha. I am on Windows, weird it says it the path is not found when I try to compile without it. It might just be my machine

Reply:
The issue is that LaTeX by default doesn't allow utilizing assets from directories up. It's a security measure. I discovered this only after I created this directory. As a workaround, I symlinked the assets dir in there, totally forgetting that it's a *nix thing. I've heard you can use symlinks on Windows nowadays, but they're not supported by default. Can you try to make it work?

Reply:
I got it. That fixed a few issues I was seeing with the build too.

Reply:
Nice, mind sharing how did you do it?

Reply:
Sure. I followed some basic instructions here: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/mklink

And being careful to commit that change to git